pkgs/top-level/all-packages.nix

```diff
@@ -4167,9 +4167,9 @@ with pkgs;
 
   bozohttpd-minimal = bozohttpd.override { minimal = true; };
 
-  brasero-original = lowPrio (callPackage ../tools/cd-dvd/brasero { });
+  brasero = callPackage ../tools/cd-dvd/brasero { };
 
-  brasero = callPackage ../tools/cd-dvd/brasero/wrapper.nix { };
+  brasero-wrapped = callPackage ../tools/cd-dvd/brasero/wrapper.nix { };
```

Comment:
.. and this just brasero. Compare with other examples in nixpkgs, like Firefox and chromium.